official/transformer/v2/metrics.py

```diff
@@ -182,24 +182,3 @@ def transformer_loss(logits, labels, smoothing, vocab_size):
                                                 vocab_size)
   return tf.reduce_sum(xentropy) / tf.reduce_sum(weights)
 
-
```

Comment:
************* Module official.transformer.v2.metrics
C:184, 0: Trailing newlines (trailing-newlines)

No idea using the github UI but I suspect using sublime or some other editor you will see what is up.